kymatio/scattering1d/backend/torch_skcuda_backend.py

```diff
@@ -182,13 +182,13 @@ def __call__(self, input, k):
           output[ty * NT + tx] = res;
         }
         '''
-        B = input.size(0) * input.size(1)
-        T = input.size(2)
-        periodize = load_kernel('periodize', kernel, B=B, T=T, k=k, dtype=get_dtype(input))
+        B = x.size(0) * x.size(1)
+        T = x.size(2)
+        periodize = load_kernel('periodize', kernel, B=B, T=T, k=k, dtype=get_dtype(x))
```

Comment:
this is not batch size independant, is it?

Reply:
Not sure, we should make a test for that.

Reply:
agreed